docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "3.8"
+services: 
+  api:
+    build: .
+    command: yarn dev
+    ports:
+      - "3000:3000"
+    volumes: 
```

Comment:
This is not useful to this Docker image, because your api service don't need to store anything.